_layouts/content-base.html

```diff
@@ -4,7 +4,14 @@
     <main class="content {% block content_modifiers -%}{%- endblock %}" id="main" role="main">
         {% block hero -%}{%- endblock %}
         <div class="content_bar"></div>
-        {% block pre_content scoped -%}{%- endblock %}
+        {% block pre_content scoped -%}
+            {% if breadcrumb_items | length > 0 %}
+                <div class="wrapper">
```

Comment:
Is `wrapper` too general? Will this be used elsewhere? Possibly rename class `breadcrumb_wrapper`